functions/FirebaseU/FirebaseUtils.js

```diff
@@ -20,21 +20,18 @@ const getID = (str) => str.substring(str.lastIndexOf('/') + 1)
 const logIn = async() => {
   var signin = await app.auth().signInWithPopup(googleProvider)
   
-  if(signin)
+  if(signin){
     document.location.href = '/explore';
+    localStorage.setItem("email", signin.additionalUserInfo.profile.email);
```

Comment:
Here, seems like you should be storing the jwt in local storage and sending the entire jwt to the back-end when making calls. Storing the email only could give us security issues.